### DISCUSSION TITLE
Pricing and Self-Hosting

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [zitadel/zitadel#8845] [Q&A] [answered]
Self Hosting only shows up under the Enterprise Pricing. Do I need to buy a licence if I self host everything and do not need support? Or do I need always a licence of some kind?

**Comment by mffap** [comment, marked as answer]
> Do I need to buy a licence if I self host everything and do not need support? 

ZITADEL is released under an open source license. You can find the license and all conditions here:
https://github.com/zitadel/zitadel/blob/main/LICENSE

> Or do I need always a licence of some kind?
That is a complicated question in the software copyright world ;) Yes you should care about the license and the condition that grants you certain rights. See the link above for more details.

**Reply by sschneider-ihre-pvs** [reply]
![grafik](https://github.com/user-attachments/assets/c78f2189-196f-4173-b253-8a4411576fd6)
I guess I was looking for something like that, thanks @mffap

**Reply by nnWhisperer** [reply]
I guess it's changed from apache to agpl recently. I guess that one's allowed to use it without having to display his source code for the remaining project, right? Or does that mean that it's now only allowed to be used for public work?

**Reply by fforootd** [reply]
Not sure I can entirely follow but during the license change we wrote down this page https://zitadel.com/license-faq

Maybe this helps 😁 otherwise let me know, maybe I can provide more detailed pointers.
